packages/cli/test/unit/cmds/beacon.test.ts

```diff
@@ -77,15 +77,6 @@ describe("cmds / beacon / args handler", () => {
     expect(peerId.toString()).toBe(prevPeerId.toString());
   });
 
-  it("Set known deposit contract", async () => {
-    const {options} = await runBeaconHandlerInit({
-      network: "mainnet",
-    });
-
-    // Okay to hardcode, since this value will never change
-    expect(options.eth1.depositContractDeployBlock).toBe(11052984);
-  });
```

Comment:
~~Are you sure this should get deleted?  Seems unrelated to the runner changing.  Is it a bad test or something?~~

NM.  I see this got moved to the other file